html-css/week1/Nina/portfolio/style.css

```
@@ -0,0 +1,376 @@
+
+
+    li {
+      display: inline-block;
+      margin-right: 10px;
+
+      a {
+        display: inline-block;
+        padding: 10px 20px;
+
+        &:hover {
+          background-color: rgba(241, 240, 240, 0.962);
+          border-radius: 10px;
+          text-decoration: none;
+        }
+
+      }
+    }
+  }
+
+  @media(max-width:768px) {
+    nav {
+      width: 100%;
+      margin-top: 10px;
+    }
+
+    .contact-me-button {
+      height: auto;
+    }
+  }
+}
+
+.button {
+  background-color: #0a61ae;
+  border-radius: 10px;
+  color: white;
+  height: 40px;
+
+  &:hover {
+    background-color: black;
+    transition: ease 0.3s all;
+  }
+
+
+}
+
+.hero {
+  display: flex;
+  max-width: 1440px;
+  gap: 20px;
+
+  @media(max-width:1024px) {
+    flex-direction: column;
+  }
+
+  margin-top: 30px;
+
+  .hero-title {
+    color: rgba(39, 102, 237, 0.719);
+    font-size: 45px;
+  }
+
+  .hero-h1 {
+    font-size: 40px;
+    line-height: 1;
+    margin-bottom: 10px;
+
+    small {
+      display: block;
+      font-weight: 50;
+    }
+  }
+
+  .hero-blue {
+    flex: 1;
+    background-color: rgba(212, 236, 255, 0.623);
+    border-radius: 30px;
+    padding: 30px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    .call-to-action {
+      margin-top: 20px;
+      margin-bottom: 10px;
+
+      a {
+        margin-right: 10px;
+        margin-bottom: 10px;
+
+      }
+    }
+
+
+  }
+
+
+  .hero-pink {
+    flex: 1;
+    background-color: #f7906868;
+    border-radius: 30px;
+    padding: 0px 30px;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+
+    img {
+      margin-top: -60px;
+      max-width: 420px;
+      margin-bottom: 0;
+    }
+
+  }
+
+  .call-to-action {
+    a {
+      padding: 10px;
+    }
+  }
+
+}
+
+@media(max-width:768px) {
+  .container {
+    padding-left: 0;
+    padding-right: 0;
+    margin-left: 10px;
+    margin-right: 10px;
+  }
+}
+
+.social-links {
+  img {
+    transition: transform 0.5s;
+
+    &:hover {
+      text-decoration: none;
+      transform: translateY(-5px);
+    }
+  }
+}
+
+h3 {
+  font-size: 32px;
+  font-weight: 600;
+  line-height: 1;
+  margin-bottom: 20px;
+  color: rgba(39, 102, 237, 0.719);
+}
+
+h2 {
+  font-size: 64px;
+  line-height: 1;
+  margin-bottom: 10px;
+  text-align: center;
+  padding: 30px;
+
+  small {
+    display: block;
+    font-weight: 100;
+    font-size: 0.5em;
+    color: var(--base);
+  }
+}
+
+.holder-blue {
+  display: flex;
+  background-color: rgba(212, 236, 255, 0.623);
+  border-radius: 30px;
+  padding: 5px 30px 30px 30px;
+  gap: 20px;
+
+  @media(max-width:768px) {
+    flex-direction: column;
+  }
+
+  .left-column {
+    flex: 1;
+  }
+
+  ul {
+    li {
+      display: inline-block;
+      padding: 10px;
+      background-color: #0a61ae;
+      border-radius: 10px;
+      color: white;
+
+      @media(max-width:768px) {
+        margin: 3px;
+      }
+
+    }
+  }
```

```suggestion
  ul  li {
    display: inline-block;
    padding: 10px;
    background-color: #0a61ae;
    border-radius: 10px;
    color: white;

    @media(max-width:768px) {
      margin: 3px;
    }
  }
```
but even better would be to use a class name.